---
title: "\"Catch\" muss mit einem entsprechenden \"End Try\" abgeschlossen werden."
ms.date: 07/20/2015
f1_keywords:
- bc30441
- vbc30441
helpviewer_keywords:
- BC30441
ms.assetid: 0e4756b4-1f29-4073-88c5-8f8c93ba6c9e
ms.openlocfilehash: e59258a763fff7fe50f3f6fa4fde7a4f064701d6
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 09/23/2020
ms.locfileid: "91089818"
---
# <a name="catch-must-end-with-a-matching-end-try"></a>"Catch" muss mit einem entsprechenden "End Try" abgeschlossen werden.

Im Code wird eine `Catch` -Anweisung ohne zugehörige `End Try` -Anweisung verwendet. `Catch` -Anweisungen müssen mit einer `End Try` -Anweisung abgeschlossen werden.  
  
 **Fehler-ID:** BC30441  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
1. Entfernen Sie die `Catch` -Anweisung.  
  
2. Fügen Sie eine `End Try` -Anweisung hinzu, um den Block zu beenden.  
  
## <a name="see-also"></a>Siehe auch

- [Try... Catch... Abschließend-Anweisung](../language-reference/statements/try-catch-finally-statement.md)
